tests/bcl-test/BCLTests/templates/iOSApp/ViewController.cs

```diff
@@ -120,7 +120,7 @@ public async override void ViewDidLoad ()
 				logger.Info ($"Xml result can be found {resultsFilePath}");
 			}
 			
-			logger.Info ($"Tests run: {runner.TotalTests} Passed: {runner.PassedTests} Inconclusive: {runner.InconclusiveTests} Failed: {runner.FailedTests} Ignored: {runner.SkippedTests}");
+			logger.Info ($"Tests run: {runner.TotalTests} Passed: {runner.PassedTests} Inconclusive: {runner.InconclusiveTests} Failed: {runner.FailedTests} Ignored: {runner.FilteredTests}");
```

Comment:
not clear inside PR but does `FilteredTests` contains both what you filter (categories) and what's ignored, e.g. `[Ignore]` ?

Reply:
Yes, it does. The other update was needed because the total number did not include the correct value when skipped.